src/session-pool.js

```diff
@@ -93,6 +94,12 @@ function SessionPool(database, options) {
    */
   this.options = extend({}, DEFAULTS, options);
 
+  if (this.options.writes > 1) {
+    throw new TypeError(
+      'write percentage should be represented as a float between 0.0 and 1.0'
```

Comment:
Just grammar stuff to match our usual throwing language.

```diff
- write percentage should be represented as a float between 0.0 and 1.0
+ Write percentage should be represented as a float between 0.0 and 1.0.
```

Reply:
Done!